src/pages/settings/key-settings/key-settings.html

```diff
@@ -1,22 +1,5 @@
-<ion-header class="bp-header">
-  <ion-navbar>
-    <ion-title>
-      <div [reveal-at-scroll-pos]="expandableHeader.headerHeight" [scrollArea]="scrollArea">
-        {{canSign ? ('Key Settings' | translate) : ('Read Only Wallets' | translate) }}
-      </div>
-    </ion-title>
-  </ion-navbar>
-</ion-header>
-
-<ion-content #scrollArea>
-  <div class="wrapper">
-    <expandable-header [scrollArea]="scrollArea" [fadeFactor]="5" [disableFade]="true" #expandableHeader>
-      <ion-toolbar class="wide-header__title">
-        <expandable-header-primary>
-          {{canSign ? ('Key Settings' | translate) : ('Read Only Wallets' | translate) }}
-        </expandable-header-primary>
-      </ion-toolbar>
-    </expandable-header>
+<wide-header-page title="{{canSign ? ('Key Settings' | translate) : ('Read Only Wallets' | translate) }}">
```

Comment:
Did you test if the wallet reorder still works correctly by making this change?

Reply:
Uhm.. nop. Fixing that now. Thanks!

Reply:
Updated. Reverted changes and added safe-bottom-area manually